download/index.md

```diff
@@ -1,91 +1,83 @@
 ---
+layout: base
 title: Download
+navgroup: download
 redirect_from:
+  - /downloads/
+  - /Downloads/
```

Comment:
remove these new redirects (we didn't have these urls before so no need to introduce it)